lib/commands/jobs/build.js

```diff
@@ -0,0 +1,28 @@
+// Login into jenkins command
+var conf = require("../../util/config");
+var logger = require("../../util/logger");
+var prompt = require('readline-sync');
+
+var triggerBuild = require("../../api/triggerBuild");
+
+exports.command = 'build <job>'
+exports.describe = 'Trigger build for Jenkins job'
+exports.builder = (yargs) => {
+    return yargs.count('verbose').alias('v', 'verbose');
```

Comment:
That should be part of the javascript styling tool - something like eslint. Going to add this soon. Any manual styling are hard to enforce and I do not want to make changes for this now until we would have automatic process in place that would fail the build.